WTWSMS/events/religious_events.txt

```diff
@@ -191,6 +199,7 @@ character_event = {
 		OR = {
 			religion = catholic
 			religion = fraticelli
+			religion = nicene
```

Comment:
No, not when you neither have the infrastructure nor the bureaucracy to do it. Note how Orthodox isn't included in the vanilla code, even if the Patriarchs could be mighty political players, and that is because this mechanic models the role of the Papacy, which didn't exist until much later. I can send you detailed references on Discord if you want to.